src/solver/optimisation/opt_gestion_des_bornes_cas_lineaire.cpp

```diff
@@ -100,6 +100,71 @@ double OPT_SommeDesPminThermiques(PROBLEME_HEBDO* ProblemeHebdo, int Pays, int P
     return (SommeDesPminThermiques);
 }
 
+void setBoundsForUnsuppliedEnergy(PROBLEME_HEBDO* ProblemeHebdo,
+                                  const int PremierPdtDeLIntervalle,
+                                  const int DernierPdtDeLIntervalle)
+{
+    // OUTPUT
+    double* Xmin = ProblemeHebdo->ProblemeAResoudre->Xmin;
+    double* Xmax = ProblemeHebdo->ProblemeAResoudre->Xmax;
+    double** AdresseOuPlacerLaValeurDesVariablesOptimisees
+      = ProblemeHebdo->ProblemeAResoudre->AdresseOuPlacerLaValeurDesVariablesOptimisees;
+
+    const bool reserveJm1 = (ProblemeHebdo->YaDeLaReserveJmoins1 == OUI_ANTARES);
+    const bool opt1
+      = (ProblemeHebdo->ProblemeAResoudre->NumeroDOptimisation == PREMIERE_OPTIMISATION);
+
+    for (int PdtHebdo = PremierPdtDeLIntervalle, PdtJour = 0; PdtHebdo < DernierPdtDeLIntervalle;
+         PdtHebdo++, PdtJour++)
+    {
+        const CORRESPONDANCES_DES_VARIABLES* CorrespondanceVarNativesVarOptim
+          = ProblemeHebdo->CorrespondanceVarNativesVarOptim[PdtJour];
+        const ALL_MUST_RUN_GENERATION* AllMustRunGeneration
+          = ProblemeHebdo->AllMustRunGeneration[PdtHebdo];
+        const CONSOMMATIONS_ABATTUES* ConsommationsAbattues
+          = ProblemeHebdo->ConsommationsAbattues[PdtHebdo];
+
+        for (int Pays = 0; Pays < ProblemeHebdo->NombreDePays; Pays++)
+        {
+            double C = ConsommationsAbattues->ConsommationAbattueDuPays[Pays];
```

Comment:
Maybe rename this variable to a more precise and readable name

Reply:
Done